src/networking/responses/branding-response.ts

```diff
@@ -3,7 +3,7 @@ import type { BrandingAppearance } from "../../entities/branding";
 export type BrandingInfoResponse = {
   app_icon: string | null;
   app_icon_webp: string | null;
-  appearance: BrandingAppearance;
+  appearance: BrandingAppearance | null;
```

Comment:
We can actually receive `null` from the server.
Eventually we were checking for nullish so everything was working, however I decided to re-align with what we have returned by the server